styles/components/Search.module.scss

```diff
@@ -28,6 +28,12 @@
 .advancedSearch {
   align-self: flex-start;
   min-width: 127px;
+  line-height: 1;
+  margin-left: auto;
```

Comment:
I noticed this issue and it seems it only happens on the homepage. I fixed this but it seems it was lost in the merge. So this is okay but I think I did it differently.